docs/about.html

```diff
@@ -0,0 +1,279 @@
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#program">Program</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#join">Join</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#sponsor">Sponsor</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#news">News</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="#home">
+          <h3 class="border-4 px-4 py-2 border-rose-700 text-rose-700 font-bold hover:bg-rose-700 hover:text-white hover:shadow-md hover:shadow-zinc-500 ease-in-out delay-200">
+            Garden Campaign
+          </h3>
+        </a>
+      </div>
+
+      <!-- Mobile Menu -->
+
+      <div class="md:hidden bg-hero-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-stone-300 px-5 fixed top-0 left-0 right-0 h-[100vh] transition translate-x-full delay-100 target:-translate-x-0" id="menu">
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px] py-3">
+          <a href="#home">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#home">
+            <i class="fa-solid fa-x fa-xl"></i>
+          </a>
+        </div>
+
+        <nav>
+          <ul class="flex flex-col gap-5 text-3xl font-bold px-6">
+            <li>
+              <a href="index.html">Home</a>
+            </li>
+
+            <li>
+              <a href="about.html">About</a>
+            </li>
+
+            <li>
+              <a href="#program">Program</a>
+            </li>
+
+            <li>
+              <a href="#join">Join</a>
+            </li>
+
+            <li>
+              <a href="#sponsor">Sponsor</a>
+            </li>
+
+            <li>
+              <a href="#news">News</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <!-- Hero Section -->
+
+      <section class="bg-hero-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-stone-300 pb-[100px] px-4 py-3 md:px-[5%] lg:px-[15%] md:py-[100px]">
+
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px]">
+          <a href="#home">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#menu">
+            <i class="fa-solid fa-bars fa-xl"></i>
+          </a>
+        </div>
+
+        <h2 class="text-3xl text-emerald-800 mb-2 md:text-center">
+          "Hello, Sharing world!"
+        </h2>
+
+        <h1 class="text-6xl font-extrabold text-emerald-800/[0.8] mb-6 md:text-8xl md:text-center lg:w-[70%] md:mx-auto lg:mb-10">
+          Garden and Flower Show 2023
+
+          <div class="hidden md:block border-b-2 border-rose-700 w-[60px] pt-4 mx-auto lg:pt-8"></div>
+        </h1>
```

Comment:
Kindly update the hero section in the home page to match the behance design better🙏

Your design:
![image](https://user-images.githubusercontent.com/37116322/233505097-334eae6e-d038-425b-ad5d-096982568449.png)

Behance design
![image](https://user-images.githubusercontent.com/37116322/233505117-76b80eba-c91a-4fab-ae95-d862dc9546fa.png)

A few things to note:
- Your design is really nice. But the project requirement state that you should stick to the design guidelines regarding colors. So please change the color of the main title and subtitle in this section to be `#f3401e`